app/controllers/namespaces_controller.rb

```diff
@@ -48,7 +48,14 @@ def create
   # PATCH/PUT /namespace/1
   # PATCH/PUT /namespace/1.json
   def update
-    change_name_description(@namespace, :namespace)
+    p = params.require(:namespace).permit(:name, :description, :team)
+
+    # This method already takes care of the authorization part.
+    change_name_description(@namespace, :namespace, p)
+
+    @team = Team.find_by(name: p[:team])
+    return if @team.nil?
```

Comment:
This prevents a namespace from being associated to a non-existing team. Am I right?

If that's the case shouldn't we return an alert message stating the team name does not exist?